src/style/themes/themes.spec.js

```diff
@@ -0,0 +1,158 @@
+  });
+
+  describe('base theme object', () => {
+    it('matches the base theme config', () => {
+      const emptyTheme = () => ({});
+      expect(mergeWithBase(emptyTheme)).toEqual(baseThemeConfig);
+    });
+  });
+
+  describe('smallBusinessTheme', () => {
+    it('contains the base theme', () => {
+      assertIsSubset(baseThemeConfig, smallBusinessTheme);
+    });
+
+    it('contains the smallBusinessTheme config', () => {
+      const smallBusinessConfig = {
+        colors: {
+          base: palette.productGreen,
+          primary: palette.productGreenShade(21),
+          secondary: palette.productGreenShade(41),
+          tertiary: palette.productGreenShade(61)
+        }
+      };
+
+      assertIsSubset(smallBusinessConfig, smallBusinessTheme);
+    });
+  });
+
+  describe('mediumBusinessTheme', () => {
+    it('contains the base theme', () => {
+      assertIsSubset(baseThemeConfig, mediumBusinessTheme);
+    });
+
+    it('contains the mediumBusinessTheme config', () => {
+      const medBusinessConfig = {
+        colors: {
+          base: palette.productBlue,
+          primary: palette.productBlueShade(3),
+          secondary: palette.productBlueShade(23),
+          tertiary: palette.productBlueShade(43)
+        }
+      };
+
+      assertIsSubset(medBusinessConfig, mediumBusinessTheme);
+    });
+  });
+
+  describe('largeBusinessTheme', () => {
+    it('contains the base theme', () => {
+      assertIsSubset(baseThemeConfig, largeBusinessTheme);
+    });
+
+    it('contains the largeBusinessTheme config', () => {
+      const largeBusinessConfig = {
+        colors: {
+          base: palette.amethyst,
+          primary: palette.amethystTint(10),
+          secondary: palette.amethystShade(10),
+          tertiary: palette.amethystShade(30)
+        }
+      };
+
+      assertIsSubset(largeBusinessConfig, largeBusinessTheme);
+    });
+  });
+
+  describe('genericTheme', () => {
+    it('contains the base theme', () => {
+      assertIsSubset(baseThemeConfig, genericTheme);
+    });
+
+    it('contains the genericTheme config', () => {
+      const genericThemeConfig = {
+        colors: {
+          base: palette.genericGreen,
+          primary: palette.genericGreenTint(15),
+          secondary: palette.genericGreenShade(35),
+          tertiary: palette.genericGreenShade(55)
+        }
+      };
+
+      assertIsSubset(genericThemeConfig, genericTheme);
+    });
+  });
+
+  describe('classicTheme', () => {
+    it('contains the base theme', () => {
+      assertIsSubset(baseThemeConfig, classicTheme);
+    });
+
+    it('contains the classicTheme config', () => {
+      const classicThemeConfig = {
+        colors: {
+          base: palette.productGreen,
+          primary: palette.productGreenShade(21),
+          secondary: palette.productGreenShade(41),
+          tertiary: palette.productGreenShade(61)
+        }
+      };
+
+      assertIsSubset(classicThemeConfig, classicTheme);
+    });
+  });
+});
```

Comment:
I would break your specs up, you can unit test each module individually and keeps your tests more concise.